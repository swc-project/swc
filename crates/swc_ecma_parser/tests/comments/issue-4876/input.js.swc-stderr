  > Leading (lo)

Advice:   > !
  | 
  |  something v5.1.0
  | 
  | more
  | 
  | else
  | 
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
  9 | // a
 10 | console.log(1);
    : ^
 11 | // b
    `----
  > Leading (lo)

Advice:   >  a
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
  9 | // a
 10 | console.log(1);
    : ^
 11 | // b
    `----
  > Trailing (hi)

Advice:   >  b
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
  9 | // a
 10 | console.log(1);
    :                ^
 11 | // b
    `----
