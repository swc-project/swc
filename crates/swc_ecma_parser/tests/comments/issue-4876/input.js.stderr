
  > Leading (lo)

Error: 
  > !
  | 
  |  something v5.1.0
  | 
  | more
  | 
  | else
  | 
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
 10 | console.log(1);
    : ^
    `----

  > Leading (lo)

Error: 
  >  a
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
 10 | console.log(1);
    : ^
    `----

  > Trailing (hi)

Error: 
  >  b
    ,-[$DIR/tests/comments/issue-4876/input.js:10:1]
 10 | console.log(1);
    :                ^
    `----
